src/wp-includes/deprecated.php

```diff
@@ -1150,13 +1150,13 @@ function comments_rss_link($link_text = 'Comments RSS') {
  * @see get_category_feed_link()
  *
  * @param bool $display
- * @param int $cat_ID
+ * @param int $cat_id
  * @return string
  */
-function get_category_rss_link($display = false, $cat_ID = 1) {
+function get_category_rss_link($display = false, $cat_id = 1) {
```

Comment:
Unfortunately, we aren't able to change the case in function definitions. Why?

PHP 8 introduced [named parameters](https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments), which are case-sensitive.

This is what would happen on sites running PHP 8 containing plugins/themes that used named parameters: https://3v4l.org/di62v